src/psyclone/psyGen.py

```diff
@@ -4797,6 +4878,28 @@ def __str__(self):
             result += str(entity)
         return result
 
+    def reference_accesses(self, var_accesses):
+        '''Get all variable access information. It combines the data from
+        the condition, if-body and (if available) else-body. This could
+        later be extended to handle cases where a variable is only written
+        in one the two branches.
```

Comment:
Almost! "in one of the two branches."

Reply:
Gee ... (frantically looking for an excuse) ... my keyboard is broken?  😊